lagom-eventstore-example/README.md

```diff
@@ -0,0 +1,99 @@
+# Lagom integration with IBM Project EventStore
+
+[IBM Project EventStore](https://www.ibm.com/us-en/marketplace/project-eventstore) is an in-memory database designed for massive structured data volumes and real-time analytics built on Apache SPARK and Apache Parquet Data Format. 
+
+This project demonstrates a simple Lagom service that includes a [Read-Side](https://www.lagomframework.com/documentation/current/java/ReadSide.html) processor which stores events of the [Persistent Entities](https://www.lagomframework.com/documentation/current/java/ReadSide.html) into Project EventStore for further analysis.
```

Comment:
[Persistent Entities] is linked to `ReadSide` doc page. Maybe a mistake?

Reply:
wrong copy/pasting.

Good catch!